feature/experimental/bgp/otg_tests/link_bandwidth_test/README.md

```diff
@@ -133,6 +133,34 @@ bandwidth communities to routes based on a prefix match.
         * prefix-set-3 is advertised to the DUT with community "link-bandwidth:100:0" set.
         * The DUT evaluates a match for "regex_nomatch_as100".  This does not match because the regex pattern does not include the link-bandwidth community type.
         * Community linkbw_2G should be added.
+
+[TODO] For each policy-definition created, run a subtest (RT-7.5.4.x-<policy_name_here>-export) to
+    * Use gnmi Set REPLACE option to attache `allow_all` using `/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/export-policy` to apply the policy on the DUT bgp neighbor to the ATE port 1.
+      *
+    * Use gnmi Set REPLACE option for:
+      * `/routing-policy/policy-definitions` to configure the policy
+      * Use `/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/export-policy`
+        to apply the policy on the DUT bgp neighbor to the ATE port 2.
+    * Verify expected communities are present in ATE port 2.
+      * Mark test as passing if Global Administartive valuee (ASN) of link-bakdwidth extended community **send by DUT** is either `23456` or ASN of DUT. 
+
+    * Expected community values for each policy
+      |              | set_linkbw_0                           | not_match_100_set_linkbw_1M |
+      | ------------ | -------------------------------------- | --------------------------- |
+      | prefix-set-1 | *DEPRECATED*                           | [none]                      |
```

Comment:
For the not_match_100_set_linkbw_1M policy, if there is a route with no communities, should the policy result in adding the link-bandwidth community?  I think yes?